pkg/controller/certificates/cleaner/cleaner.go

```diff
@@ -172,7 +172,7 @@ func isIssuedPastDeadline(csr *capi.CertificateSigningRequest) bool {
 
 // isOlderThan checks that t is a non-zero time after time.Now() + d.
 func isOlderThan(t metav1.Time, d time.Duration) bool {
-	return !t.IsZero() && t.Sub(time.Now()) < -1*d
+	return !t.IsZero() && time.Until(t.Time) < -1*d
```

Comment:
wouldn't this be cleaner as  `time.Since(t.Time) > d` or `time.Now() > t.Add(d)` ?

This formulation is bizarre and absent any comment indicating WHY

Reply:
`time.Since(t.Time) > d` or `time.Now() > t.Add(d)` is better. I can fix it. Thanks.